src/app/signers/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { SIGNER_KEY_MAP } from './consts';
+
+export function getSignerKeyName(signerKey: string) {
```

Comment:
While I understand the importance of unit tests in ensuring code reliability, I believe that in this particular case, a unit test may not significantly enhance our confidence in the code. The getSignerKeyName function performs a straightforward table lookup using the SIGNER_KEY_MAP constant. Given its simplicity, the function's reliability is inherently high.

Reply:
It's indeed a simple function, but I think a unit test will still help ensure that future changes do not break functionality and increase confidence during refactoring. Another issue is that it can be tricky to draw the line between which functions are 'simple' enough to not require a unit test. If we decide to not add unit tests for some functions, we might need to adjust our coverage goals. Happy to chat about this on our next sync.

Reply:
What changes would break this?
It's checking for keys in a map, with a catchall...
We shouldn't be looking at our code for places to add unit tests just to add unit tests and meet an arbitrary coverage goals. Tests take time to not only to write but run and maintain. We should only be adding unit tests when they meaningfully improve our confidence in the code we ship.
We wanted to rethink our testing strategy earlier, but we never got around to it. I think we need to think about higher levels tests more than unit tests imo. We should prioritze doing this sooner rather than later